src/components/PostDetails.tsx

```diff
@@ -2,105 +1,92 @@
+import React, { useEffect, useState } from 'react';
+import * as commentService from '../api/comments';
 import { Loader } from './Loader';
+import { CommentList } from './CommentList';
 import { NewCommentForm } from './NewCommentForm';
+import { Post } from '../types/Post';
+
+type Props = {
+  selestedPost: Post;
+};
+
+export const PostDetails: React.FC<Props> = ({ selestedPost }) => {
+  const [comments, setComments] = useState<Comment[]>([]);
+  const [isCommentsLoading, setIsCommentsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+  const [isCommentFormShown, setIsCommentFormShown] = useState(false);
+
+  useEffect(() => {
+    setIsCommentsLoading(true);
+    commentService
+      .getComments(selestedPost.id)
```

Comment:
The typo in `selestedPost` should be corrected to `selectedPost` to match the prop name.